pelops/features/hog.py

```diff
@@ -20,14 +21,38 @@ def produce_features(self, chip):
         img = self.get_image(chip)
         img = img.resize(self.image_size, Image.BICUBIC)
         img_x, img_y = img.size
+
+        # Calculate histogram of each channel
+        channels = img.split()
+        hist_features = np.zeros(3 * self.histogram_bins_per_channel)
+
+        # We expect RGB images. If something else is passed warn the user and
+        # continue.
+        if len(channels) < 3:
+            print("Non-RBG image! Vector will be padded with -1!")
```

Comment:
No, you caught me. I lied to the user. :-1: Fixed to initialize the array at -1.